portfolio.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jess Hamby | Portfolio</title>
+    <link rel="stylesheet" href="normalize.css" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body id="body">
+	<nav>
+
+		
+		<a href="#body">Jess Hamby</a>
+		<div class="nav-links">
+			<a href="#about">about me</a>
+			<a href="#portfolio">portfolio</a>
+			<a href="#contact">contact</a>
+		</div>
+	</nav>
+	<main>
+		<!-- Header -->
+		<header>
+			<img src="https://images.pexels.com/photos/1840101/pexels-photo-1840101.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="A picture of Jess Hamby" />
+			<h1>Hi, I'm Jess...</h1>
+		</header>
+		
+		<!--About -->
+		<section class="about" id="about">
+			<div class="about-text">
+				<h2>About me</h2>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean varius ac quam pharetra accumsan. Suspendisse facilisis malesuada suscipit. Vivamus lacus massa, facilisis sit amet consequat ut, rhoncus sed leo. Proin in nisl in ipsum mattis placerat. Proin eu risus bibendum nisl hendrerit bibendum. Duis posuere nisl non purus rutrum finibus. Nulla mollis egestas magna, non ultrices eros egestas a. Nunc id tortor nunc.</p>
+			</div>
+			<div class="about-img">
+				<img src="https://images.pexels.com/photos/52576/pexels-photo-52576.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Cafe scene" />
+			</div>
+		</section>
+		
+		<!-- Portfolio -->
+		<section class="portfolio" id="portfolio">
+			
+			<!-- Project 1-->
+			<h2>Portfolio</h2>
+			<section>
+				<div>
+					<h3>Project Name</h3>
+					<img src="https://images.pexels.com/photos/1229102/pexels-photo-1229102.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="View up fropm the forest floor" class="portfolio-img" />
+				</div>
+				<div class="portfolio-text">
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean varius ac quam pharetra accumsan. Suspendisse facilisis malesuada suscipit. Vivamus lacus massa, facilisis sit amet consequat ut, rhoncus sed leo.</p>
+					<br />
```

Comment:
This is technically improper use of a br tag, it should only be used to add line breaks to text blocks. The good news is you don't really need it.

Would suggest removing it, and adding some margin top to the portfilio-links, or some margin bottom to the p

Reply:
Ahhh so they're usually nested within p tags or other text?